src/material-experimental/mdc-core/option/_option-theme.scss

```diff
@@ -55,10 +55,12 @@
       theming.get-typography-config($config-or-theme));
 
   @include mdc-helpers.mat-using-mdc-typography($config) {
-    // MDC uses the `subtitle1` level for list items, but the spec shows `body1` as the correct
-    // level. Class is repeated for increased specificity.
-    .mat-mdc-option .mdc-list-item__primary-text {
-      @include mdc-typography.typography(body1, $query: mdc-helpers.$mat-typography-styles-query);
+    @if (typography.private-typography-is-2014-config($config)) {
```

Comment:
I think this is backwards, we actually want to override if it's 2018 config

Reply:
But that'll keep it as it is right now for the majority of cases.